README.md

```diff
@@ -82,6 +92,18 @@ describe('Acceptance: ember generate and destroy my-blueprint', function() {
 or more explicitly:
 
 ```js
+const { 
+  setupTestHooks,
+  emberNew,
+  emberGenerate,
+  emberDestroy,
+} = require('ember-cli-blueprint-test-helpers/helpers');
+
+const {
+  expect,
+  file,
```

Comment:
I couldn't for the life of me understood that I needed to import "file" from chai, so maybe this can help some people.